build.sh

```diff
@@ -395,14 +401,18 @@ case "$COMMAND" in
         ;;
 
     "start-object-server")
-        # kill any object servers that are still running
-        (pgrep -f realm-object-server || true) | while read pid; do
-            kill $pid
-        done
+        kill_object_server
         ./sync/start-object-server.command
         exit 0
         ;;
 
+    "reset-object-server-between-tests")
+        kill_object_server
+        # Leave the server files alone to avoid 'bad_server_ident' errors
```

Comment:
Taking a step back, why is it necessary to kill and respawn the object server between each test now that credentials and realm URLs are unique for each test?